functors task (1)/student/src/impl.cpp

```diff
@@ -1,2 +1,22 @@
 #include "impl.h"
 
+BeerOrganizer::BeerOrganizer()
+   : currentBeerBand((int)BeerBrand::None)
```

Comment:
you can avoid this constructor and initialize in-place
**never use c-cast in c++ code**